packages/compiler/test/server/folding.test.ts

```diff
@@ -0,0 +1,85 @@
+import { deepStrictEqual } from "assert";
+import { FoldingRange } from "vscode-languageserver/node.js";
+import { createTestServerHost } from "../../testing/test-server-host.js";
+
+describe("compiler: server: foldingRange", () => {
+  it("testing for foldingRange with doc", async () => {
```

Comment:
nit: that string should be read as a description of the behavior of the test. Like `it include decorator in folding range`